frontend/app/page.tsx

```diff
@@ -79,47 +96,145 @@ export default function Home() {
+        <div>
+          <form action="" className="flex flex-col gap-4">
+            <label htmlFor="">Upload your Resume in DOCX or PDF format:</label>
+            {loadingBrowse && "Uploading..."}
+            <input
+              type="file"
+              onChange={handleFileUpload}
+              disabled={loadingBrowse}
+              className="file-input file-input-bordered w-full max-w-xs"
+            />
+            {uploadData && (
+              <p className="container mx-auto px-4 py-8 text-yellow-700">
+                {uploadData}
+              </p>
+            )}
+          </form>
+        </div>
+      </section>
 
-      {/* Job Description */}
-      <div className='container mx-auto px-4 pb-6'>
+      <section
+        id="about"
+        className="flex flex-col justify-center h-screen gap-6 w-2/3">
+        <h1 className="text-6xl font-bold mb-4 text-center mt-28">About</h1>
+        <div className="text-xl">
+          <p className="p-2">
+            Welcome to Resumivise 🏢 your go-to platform for crafting a standout
+            resume! We understand the challenges of job hunting, and that is why
+            we are here to lend a helping hand.
+          </p>
+          <p className="p-2">
+            Ever wondered how well your resume aligns with a job description?
+            Our unique feature does just that! We compare your resume to the job
+            requirements, giving you a percentage match. But we do not stop
+            there we guide you on enhancing your strengths and addressing any
+            flaws in your resume. Think of us as your personal resume
+            consultant!
+          </p>
+          <p className="p-2">
+            Worried about privacy? We have got you covered. Your data is
+            precious, and we treat it that way. No need to fret about your
+            information lingering in the cloud. Your uploaded resume is promptly
+            removed after processing. Plus, you have control the JSON data file,
+            holding extracted info from your resume, can be wiped manually at
+            your request or automatically after a few hours.
+          </p>
+          <p className="p-2">
+            Still have privacy concerns? Check out our app repository on
+            <a href="https://github.com/PooriaT/Resumivise">GitHub</a>
+            for transparency and peace of mind. Your concerns matter to us!
+          </p>
+          <p className="p-2 text-red-500">
+            Disclaimer: While Resumivise is here to assist, please note that we
+            do not take responsibility for any issues that may arise in your
+            resume during the application process. It is crucial to review your
+            resume thoroughly before hitting that apply button.
+          </p>
+          <p className="p-2">
+            Ready to take the next step in your career? It is time to apply with
+            confidence, thanks to Resumivise!
+          </p>
```

Comment:
At the bottom of the About section, I would like to add a part related to team members which refers to my LinkedIn profile and yours—something like a thumbnail of our LinkedIn profiles.  Also, please add the Contact Us part, the same as what is available on the live version of the application. Only provide the email address.